solidity/dashboard/src/css/liquidity-page.less

```diff
@@ -46,6 +46,15 @@
     }
   }
 
+  .liquidity__info--locked {
+    .liquidity__info-tile {
+      &:extend(.bg-grey-20);
+      .liquidity__info-tile__title {
+        color: @color-grey-60!important;
```

Comment:
[Loading skeletons](https://github.com/keep-network/keep-core/blob/master/solidity/dashboard/src/components/LiquidityRewardCard.jsx#L79) in `mint-20` color on the `grey-60` backround doens't look good. We should change the color of the loading skeleton.

Reply:
I just noticed that too and i definitely agree. I will change that